### ISSUE TITLE
installer fails because openshift-sdn is completely different

### ISSUE THREAD

**Issue by thoraxe** [detiber/openshift-ansible#31]
```
TASK: [openshift_sdn_master | Install openshift-sdn-master] ******************* 
failed: [ose3-master.example.com] => {"changed": false, "failed": true, "rc": 0, "results": []}
msg: No Package matching 'openshift-sdn-master' found available, installed or updated

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/root/config.retry

localhost                  : ok=1    changed=0    unreachable=0    failed=0   
ose3-master.example.com    : ok=35   changed=14   unreachable=0    failed=1   

[root@ose3-master openshift-ansible]# yum list '*openshift-sdn*'
Loaded plugins: product-id, subscription-manager
Installed Packages
openshift-sdn-ovs.x86_64                                                                      0.5.2.0-0.git.176.c386339.el7ose                                                                       @RH7-RHOSE-3.0
[root@ose3-master openshift-ansible]# 
```

There appears to no longer be an sdn-master/sdn-node package?

**Comment by sdodson** [comment]
Yeah, we need to refactor the sdn stuff considerably. For now set openshift_use_openshift_sdn = false in /etc/ansible/hosts then follow the directions described at https://github.com/openshift/origin/pull/2375#issue-78550523

**Comment by detiber** [comment]
I'm getting close to wrapping up the config templating changes (with the new sdn configs), Once I've finished ironing out the issues while setting up the summit env, I'll polish it up and get it merged into the v3-beta4 branch to unblock testing.
